publish_feed.py

```diff
@@ -2,39 +2,44 @@
 # YOU MUST INSTALL ATPROTO SDK
 # pip3 install atproto
 
+import os
+
+from dotenv import load_dotenv
 from atproto import Client, models
 
+load_dotenv()
+
 # YOUR bluesky handle
 # Ex: user.bsky.social
-HANDLE: str = ''
+HANDLE: str = os.environ.get("HANDLE")
```

Comment:
Could we use a single quote mark here for the rest of the project?

Reply:
Done